schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -470,6 +470,42 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ApiManagement.json#/resourceDefinitions/service_tags" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ApiManagement.json#/resourceDefinitions/service_templates" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ApiManagement.json#/resourceDefinitions/service_users" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-01-01/Microsoft.ApiManagement.json#/resourceDefinitions/service" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2019-01-01/Microsoft.ApiManagement.json#/resourceDefinitions/service" }, [](start = 18, length = 127)

we have a new root schema now: 2019-04-01/deploymentTemplate.json. Can you please add them to the new schema as well?